src/fundus/publishers/uk/the_sun.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class TheSunParser(ParserProxy):
+    class V1(BaseParser):
+        _summary_selector = CSSSelector("div[data-gu-name='standfirst'] p")
+        _paragraph_selector = CSSSelector("div.article__content > p")
```

Comment:
There are some articles that also contain subheadlines, such as this one: https://www.thesun.co.uk/betting/21748039/best-monopoly-live-casinos/. It would be great, if you could also add a subheadline selector

Reply:
As requested, I added a subheadline selector and succesfully executed `python -m scripts.generate_parser_test_files -p TheSun -o`

Reply:
I also black, isort, mypy and pytest. All of them passed on my local machine without any erros :)

Reply:
Perfect, thanks a lot :)

Reply:
Did I change everything that was requested or did I miss something :)?

Reply:
I think I picked the wrong subheadline. I changed the subheadline selector and re-generated test files, executed pytest, black, isort and mypy. Pycharm tells me there are no file changes thus I can't push or commit test.

Reply:
I just reviewed it and judging by the subheadline selector you chose, I think you got the correct idea. A subheadline in Fundus a line of text separating paragraphs into logical entities. For example in https://www.thesun.co.uk/news/27470413/ukraine-torpedo-submarine-black-sea-battle/ `CAN IT BE REAL?` would be considered a subheadline. In this case I would suggest something like this as the subheadline selector: `div.article__content > h2.wp-block-heading`